components/chainhook-sdk/src/observer/http.rs

```diff
@@ -178,6 +178,25 @@ pub fn handle_new_stacks_block(
     success_response()
 }
 
+#[cfg(feature = "stacks-signers")]
+#[post("/stackerdb_chunks", format = "application/json", data = "<payload>")]
+pub fn handle_stackerdb_chunks(
```

Comment:
Is this function the best time to grab `SystemTime::now()`? Or is the receipt time already handled elsewhere?